src/test/scala/chiselTests/naming/NamePluginSpec.scala

```diff
@@ -75,14 +75,12 @@ class NamePluginSpec extends ChiselFlatSpec with Utils {
       val foo, bar = IO(Input(UInt(8.W)))
 
       {
-        val x1 = chisel3.assert(1.U === 1.U)
```

Comment:
Deleted because a label doesn't make sense for chisel3.assert

Reply:
The problem is that chisel3.assert never use plain FIRRTL assert. IfElseFatal style assertion doesn't have a label and we have ignored labels. I think it's fair to add a new API for emitting a plain FIRRTL assert and use that API here though.

Reply:
You're saying it never had a label, but if it didn't, how did this test pass previously?

Reply:
chisel3.assert emitted an assert op along with print op. The assert op is actually a dummy op deleted by a parser and this test passed because the dummy assert had a label.